src/frontend/src/scheduler/error.rs

```diff
@@ -16,12 +16,10 @@ use risingwave_common::error::{ErrorCode, RwError, TrackingIssue};
 use risingwave_rpc_client::error::RpcError;
 use thiserror::Error;
 
-use crate::scheduler::plan_fragmenter::QueryId;
-
 #[derive(Error, Debug)]
 pub enum SchedulerError {
-    #[error("Pin snapshot error: {0} fails to get epoch {1}")]
-    PinSnapshot(QueryId, u64),
+    #[error("Pin snapshot error: fails to get epoch {0}, max retry reahced {1}")]
+    PinSnapshot(u64, u64),
```

Comment:
Why change this?

Reply:
The original is PinSnapshot(QueryID,epoch)
We don't need it now because when we Pin snapshot we don't know the QueryID. 
I change it and reuse it in pin_epoch_with_retry()